reporter/src/main/resources/templates/asciidoc/pdf-theme.yml

```diff
@@ -0,0 +1,4 @@
+extends: default
+role:
+  alert:
+    font-color: #ff0000
```

Comment:
Consider a "friendlier" color like #cc0000.

Reply:
I was aligning with

https://github.com/oss-review-toolkit/ort/blob/9c48a8274b4196cc74b3929c3e620e86aa9fea5d/examples/asciidoctor-pdf-theme.yml#L31-L33

but I'm planning to remove that example file later anyway, so I'll change the color here.